operators/hpe-csi-operator/destinations/certified-operators/current-version/manifests/hpe-csi-operator.clusterserviceversion.yaml

```diff
@@ -30,31 +50,57 @@ metadata:
               "nimble": false,
               "primera": false
             },
+            "disableHostDeletion": false,
             "disableNodeConfiguration": false,
             "disableNodeConformance": false,
             "disableNodeGetVolumeStats": false,
+            "disableNodeMonitor": false,
             "imagePullPolicy": "IfNotPresent",
+            "images": {
+              "csiAttacher": "registry.k8s.io/sig-storage/csi-attacher@sha256:b4d611100ece2f9bc980d1cb19c2285b8868da261e3b1ee8f45448ab5512ab94",
+              "csiControllerDriver": "registry.connect.redhat.com/hpestorage/csi-driver@sha256:d96e2a63029267abcb3eaf032eb70401861866d636bdb6f192ceb6cee954bb67",
+              "csiExtensions": "registry.connect.redhat.com/hpestorage/csi-extensions@sha256:c578d8b4075278fd03bab77b0969585d55a2dac9558bc45a01ba0828834882f2",
+              "csiNodeDriver": "registry.connect.redhat.com/hpestorage/csi-driver@sha256:d96e2a63029267abcb3eaf032eb70401861866d636bdb6f192ceb6cee954bb67",
+              "csiNodeDriverRegistrar": "registry.k8s.io/sig-storage/csi-node-driver-registrar@sha256:f25af73ee708ff9c82595ae99493cdef9295bd96953366cddf36305f82555dac",
+              "csiProvisioner": "registry.k8s.io/sig-storage/csi-provisioner@sha256:405a14e1aa702f7ea133cea459e8395fe40a6125c088c55569e696d48e1bd385",
+              "csiResizer": "registry.k8s.io/sig-storage/csi-resizer@sha256:a541e6cc2d8b011bb21b1d4ffec6b090e85270cce6276ee302d86153eec0af43",
+              "csiSnapshotter": "registry.k8s.io/sig-storage/csi-snapshotter@sha256:2e04046334baf9be425bb0fa1d04c2d1720d770825eedbdbcdb10d430da4ad8c",
+              "csiVolumeGroupProvisioner": "registry.connect.redhat.com/hpestorage/volume-group-provisioner@sha256:51a49a6848e7c22c4c4307496724c6260497aa93d70858b44c16972e12fd18f1",
+              "csiVolumeGroupSnapshotter": "registry.connect.redhat.com/hpestorage/volume-group-snapshotter@sha256:39914c29b33dcb11b5df8d15b5c90913bcf1c994b44d7f202b8d1cb52e9919a9",
+              "csiVolumeMutator": "registry.connect.redhat.com/hpestorage/volume-mutator@sha256:57a4449afc2c3e31586e6fbb760f6e2503abcaa1baca70c839e039ea66534135",
+              "nfsProvisioner": "registry.connect.redhat.com/hpestorage/nfs-provisioner@sha256:c15b5e76c58cb3f75f81d5e60f61baddf891fa8c49e2f6f580226f129413cd07",
+              "nimbleCSP": "registry.connect.redhat.com/hpestorage/alletra-6000-and-nimble-csp@sha256:ab4003f93032129e7f5768dd9b2b047fbfd435b8e0f03e4a60d6d30693320758",
+              "primera3parCSP": "registry.connect.redhat.com/hpestorage/alletra-9000-primera-and-3par-csp@sha256:320c335062338378889c1dde2ecba955d3c4d6f4fab120ea075bc4a6c5ffe7af"
+            },
```

Comment:
All these images will present in the redhat registry now ?

Reply:
Good catch! They're not, this is a bug in the Makefile, I'll update.